CHANGELOG.md

```diff
@@ -10,6 +10,38 @@ to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ### Breaking changes
 
+### Deprecations
+
+### New features
+
+### Fixed bugs
+
+## [0.35.0] - 2025-11-05
+
+### Release highlights
+
+* The new `jj git colocation enable` and `jj git colocation disable` commands
+  allow converting between colocated and non-colocated repositories.
+
+* Tags can now be created, updated, and deleted using `jj tag set` and
+  `jj tag delete`. Updated tags are currently always exported to Git as
```

Comment:
nit: "Updated" -> "Created/updated"? Especially since we made the distinction in the previous sentence.